cmd/lambda/leases/main.go

```diff
@@ -128,13 +128,10 @@ func initConfig() {
 	}
 
 	// load up the values into the various settings...
-	err := cfgBldr.WithEnv("AWS_CURRENT_REGION", "AWS_CURRENT_REGION", "us-east-1").Build()
-	if err != nil {
-		log.Printf("Error: %+v", err)
-	}
+	_ = cfgBldr.WithEnv("AWS_CURRENT_REGION", "AWS_CURRENT_REGION", "us-east-1").Build()
```

Comment:
this was actually a mistake in the code here -- `WithEnv` returns the `ConfigurationBuilder` object, not an error (so this would have logged an error message with every run)